pulsar-client/src/main/java/com/yahoo/pulsar/client/impl/ConsumerImpl.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright 2016 Yahoo Inc.
- *
+ * <p>
```

Comment:
This additional characters are making the license checker to fail the validation :)